lib/ug.js

```diff
@@ -41,6 +55,8 @@ module.exports = {
 
     const [, prefix, suffix] = matching
     const groupName = `ladok2.kurser.${prefix}.${suffix}.antagna_${term}.${round}`
+
+    log.trace(`UG: Searching members of group "${groupName}"`)
```

Comment:
These log.trace, will there be lot's of such log lines? I'm thinking that they might fill up our log server even if we normally don't read trace logs

Reply:
No. By default only log lines above "info" are presented in graylog (neither `debug`, `trace` can be seen). However when running the app locally it might be good to have them